6.recursiveSum.js

```diff
@@ -9,9 +9,11 @@
 
 
 const recursiveSum = (arr) => {
-
+    if (arr.length === 0) return;
+    if (arr.length === 1) return arr[0];
+    let sum = 0
```

Comment:
if sum is always 0 just ignore it. 
return arr.pop() + recuSum 

This problem however is asking for a pure function so you shouldn't mutate the array with pop. Make an i variable that updates with each recursive call.